__test__/TestUtils.js

```diff
@@ -1,75 +1,157 @@
-
-
 // Stolen from https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
```

Comment:
I probably didn't mean this exact wording to make it into the final thing... 😟  A more professional sounding attribution might be needed... 😊

Reply:
IIRC I used that wording elsewhere and it’s been copy-pasted around.

Reply:
Ahh good, it just sounded like something I would write, though I didn't remember writing it... Explains why!